src/makeCalculator.js

```diff
@@ -37,7 +37,31 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+    operate(callback, number) {
+      this.result = callback(number, this.result);
```

Comment:
It would be much better to bind the `callback` function to the context via the `call` method, for example to access this (in this case this object)